modules/qualitycontrol/QualityController.ts

```diff
@@ -159,6 +161,26 @@ export class QualityController {
             (tpc: TraceablePeerConnection, stats: Map<number, IOutboundRtpStats>) => this._processOutboundRtpStats(tpc, stats));
     }
 
+    /**
+     * Creates a debounced function that delays the execution of the provided function until after the specified delay
```

Comment:
Isn't this basically lodash's throtlle? https://lodash.com/docs/4.17.15#throttle

Reply:
Wait, I see you imported it but then didn't use it?

Reply:
Oh yes, I was testing with lodash throttle with both the leading and trailing options set to true and ran into this limitation - If leading and trailing options are true, func is invoked on the trailing edge of the timeout only if the throttled function is invoked more than once during the wait timeout. 
I can go back and test lodash throttle with only the trailing option set to true and I believe the end result will be the same. Will update this.

Reply:
Actually when I pass `{ leading: false, trailing: true }` to debounce's throttle, my testcases fail, its missing some of the calls when we expect them.

Reply:
We are waiting for 1000 and advancing by 1000 I bet there are shenanigans there? Perhaps we should test by waiting a bit longer?

Reply:
Ok let's have it this way 👍